packages/xy-chart/src/selection/Brush.jsx

```diff
@@ -69,14 +70,22 @@ const defaultProps = {
   disableDraggingSelection: false,
   onMouseMove: null,
   onMouseLeave: null,
+  onClick: null,
 };
 
 class Brush extends React.Component {
   constructor(props) {
     super(props);
+
+    this.BaseBrush = React.createRef();
```

Comment:
I've never used this method before! I think this miiight mean that we would no longer be react 15 compatible. (which might be okay?!)